source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -5618,6 +5664,31 @@ export class ColumnSet extends StylableCardElementContainer {
     private _columns: Array<Column> = [];
     private _renderedColumns: Array<Column>;
 
+    private createColumnInstance(json: any, errors: Array<HostConfig.IValidationError>): Column {
+        return createCardObjectInstance<Column>(
+            this,
+            json,
+            [], // Forbidden types not supported for elements for now
```

Comment:
> for no [](start = 61, length = 7)

Curious: Why "for now" ? Is the concept of forbidden types a temporary idea?

Reply:
There is no need to support it for elements right now. The underlying logic (createCardObjectInstance) supports it but there is no requirement to use it at this time for elements.